src/api/queries.js

```diff
@@ -34,17 +42,17 @@ exports.getQuestionnaire = `
               ...answerFragment
               ... on BasicAnswer {
                 secondaryLabel
+                parentAnswerId
               }
               ... on MultipleChoiceAnswer {
                 options {
-                  id
-                  label
-                  description
-                  value
-                  qCode
+                  ...optionFragment
                 }
-                otherAnswer {
-                  ...answerFragment
+                other {
+                  option {
+                    ...optionFragment
+                    otherAnswerId 
```

Comment:
Similarly, `otherAnswerId` isn't a field that can be retrieved when querying for an Option.

This can be changed to something like this to get both the Answer and Option.

```
... on MultipleChoiceAnswer {
    options {
      ...optionFragment
    }
    other {
      answer {
        ...answerFragment
      }
      option {
        ...optionFragment
      }
    }
  }
```